process/smartContract/process.go

```diff
@@ -160,12 +163,42 @@ func (sc *scProcessor) ExecuteSmartContractTransaction(
 	tx data.TransactionHandler,
 	acntSnd, acntDst state.UserAccountHandler,
 ) (vmcommon.ReturnCode, error) {
-	defer sc.tempAccounts.CleanTempAccounts()
-
 	if check.IfNil(tx) {
 		return 0, process.ErrNilTransaction
 	}
-	log.Trace("scProcessor.ExecuteSmartContractTransaction()", "sc", tx.GetRcvAddr(), "data", string(tx.GetData()))
+
+	sw := sc.monitorBeginExecuteSmartContractTransaction()
+	returnCode, err := sc.doExecuteSmartContractTransaction(tx, acntSnd, acntDst)
+	sc.monitorEndExecuteSmartContractTransaction(sw, tx, returnCode, err)
+	return returnCode, err
+}
+
+func (sc *scProcessor) monitorBeginExecuteSmartContractTransaction() *core.StopWatch {
+	sw := core.NewStopWatch()
```

Comment:
This is a little overcomplicated. There is no need for this function

Reply:
Fixed, removed the extra 2 functions.